src/Home/Home.tsx

```diff
@@ -24,6 +24,10 @@ import { GlobalText } from "../components/GlobalText"
 import { Button } from "../components/Button"
 import { isPlatformiOS } from "../utils/index"
 import { getLocalNames } from "../locales/languages"
+import { useConfigurationContext } from "../ConfigurationContext"
+import { useBluetoothStatus } from "./useBluetoothStatus"
```

Comment:
These three useThingStatus custom hooks are jumping out to me like they may want to be pulled into a common context to manage the ActivationState of the app.  I wouldnt make any changes to this pr, but wondering about your thoughts?

Reply:
I like that. Seems like a natural grouping. Would be fun to pair on.